src/server.ts

```diff
@@ -0,0 +1,140 @@
+import path from 'path';
+import http, { IncomingMessage, ServerResponse } from 'http';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import importBenchmarks from './rollup-plugin-import-benchmarks';
+import ts from 'typescript';
+import * as glob from 'glob';
+
+import * as staticAlias from 'node-static';
+
+import * as rollup from 'rollup';
+
+import alias from '@rollup/plugin-alias';
+
+export default function runServer({ port = 8080 } = { port: 8080 }) {
+	const cwd = process.cwd();
+
+	const builtFileServer = new staticAlias.Server(`${__dirname}/../lib`);
+	const performanceFileServer = new staticAlias.Server(`${__dirname}/../src`);
+	const rootFileServer = new staticAlias.Server(`${cwd}`);
+	const modulesFileServer = new staticAlias.Server(`${cwd}/node_modules`);
+
+	// First: start TSC in watch mode, compiling performance/**/*.ts to some tmp folder
+	// Then: use rollup to bundle that into one file, again in watch mode
+	// Then: host that file
+
+	const fileNames = glob.sync(`${__dirname}/**/*.ts`);
+	const performanceTestFiles = glob.sync(`${cwd}/performance/**/*.benchmark.ts`);
+
+	console.log('server starting');
+
+	// TODO: do this in a watch
```

Comment:
Do we need to fix this? If so, would using a TS rollup plugin (instead of separate TS calls) work?